README.md

```diff
@@ -141,11 +143,13 @@ When ingesting a metric, the following actions occur:
 
 ### Data Query
 
-The goals of a data query are
-- retrieve the data for the requested tenant, metric name, and one or more tags
-- provide the timestamped values that fall within the requested time range
+The goal of a data query is to provide the data for the requested tenant, metric name, and one or more tags whose timestamped values fall within the requested time range
+
+Such a query is performed in two steps:
+- query the series_set table to retrieve the data for the requested tenant, metric name, and tags 
```

Comment:
Just to avoid confusion with the next bullet
```suggestion
- query the series_set table to retrieve the metadata for the requested tenant, metric name, and tags 
```